extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/ArrowToTableConverter.java

```diff
@@ -70,7 +69,7 @@ private static BarrageProtoUtil.MessageInfo parseArrowIpcMessage(final byte[] ip
     }
 
     @ScriptApi
-    public synchronized void setSchema(final byte[] ipcMessage) {
+    public synchronized void setSchema(final ByteBuffer ipcMessage) {
```

Comment:
So, I think we need to add a big warning here and link to https://github.com/jpy-consortium/jpy/issues/126; no strong references to this buffer can exist by the end of this method. I think we need to extend this warning into the implementation of parseSchema as well - it *can't* hold any references to `Schema header` (since that has references to the byte buffer).